_utils/terror_gen/errors_release.txt

```diff
@@ -273,14 +273,14 @@ ErrSyncerUnitOnlineDDLSchemeNotSupport,[code=36040:class=sync-unit:scope=interna
 ErrSyncerUnitOnlineDDLOnMultipleTable,[code=36041:class=sync-unit:scope=internal:level=high],"online ddl changes on multiple table: %s not supported"
 ErrSyncerUnitGhostApplyEmptyTable,[code=36042:class=sync-unit:scope=internal:level=high],"empty tables not valid"
 ErrSyncerUnitGhostRenameTableNotValid,[code=36043:class=sync-unit:scope=internal:level=high],"tables should contain old and new table name"
-ErrSyncerUnitGhostRenameToGhostTable,[code=36044:class=sync-unit:scope=internal:level=high],"rename table to ghost table %s not supported"
-ErrSyncerUnitGhostRenameGhostTblToOther,[code=36045:class=sync-unit:scope=internal:level=high],"rename ghost table to other ghost table %s not supported"
-ErrSyncerUnitGhostOnlineDDLOnGhostTbl,[code=36046:class=sync-unit:scope=internal:level=high],"online ddls on ghost table `%s`.`%s`"
+ErrSyncerUnitGhostRenameToGhostTable,[code=36044:class=sync-unit:scope=internal:level=high],"rename table to gh-ost temporary table %s not supported"
+ErrSyncerUnitGhostRenameGhostTblToOther,[code=36045:class=sync-unit:scope=internal:level=high],"rename gh-ost temporary table to other temporary table %s not supported"
+ErrSyncerUnitGhostOnlineDDLOnGhostTbl,[code=36046:class=sync-unit:scope=internal:level=high],"online ddl metadata for table `%s`.`%s` not found"
 ErrSyncerUnitPTApplyEmptyTable,[code=36047:class=sync-unit:scope=internal:level=high],"empty tables not valid"
 ErrSyncerUnitPTRenameTableNotValid,[code=36048:class=sync-unit:scope=internal:level=high],"tables should contain old and new table name"
-ErrSyncerUnitPTRenameToGhostTable,[code=36049:class=sync-unit:scope=internal:level=high],"rename table to ghost table %s not supported"
-ErrSyncerUnitPTRenameGhostTblToOther,[code=36050:class=sync-unit:scope=internal:level=high],"rename ghost table to other ghost table %s not supported"
-ErrSyncerUnitPTOnlineDDLOnGhostTbl,[code=36051:class=sync-unit:scope=internal:level=high],"online ddls on ghost table `%s`.`%s`"
+ErrSyncerUnitPTRenameToPTTable,[code=36049:class=sync-unit:scope=internal:level=high],"rename table to pt temporary table %s not supported"
+ErrSyncerUnitPTRenamePTTblToOther,[code=36050:class=sync-unit:scope=internal:level=high],"rename pt temporary table to other temporary table %s not supported"
+ErrSyncerUnitPTOnlineDDLOnPTTbl,[code=36051:class=sync-unit:scope=internal:level=high],"online ddl metadata for table `%s`.`%s` not found"
```

Comment:
the error log  same with `ErrSyncerUnitGhostOnlineDDLOnGhostTbl`, maybe we can only use one or add `gh-ost` and `pt` in error log

Reply:
address in e1ba8ba